django/core/management/commands/migrate.py

```diff
@@ -227,8 +227,9 @@ def handle(self, *args, **options):
                 changes = autodetector.changes(graph=executor.loader.graph)
                 if changes:
                     self.stdout.write(self.style.NOTICE(
-                        "  Your models have changes that are not yet reflected "
-                        "in a migration, and so won't be applied."
+                        "  Your models in app(s): %s have changes that are not "
+                        "yet reflected in a migration, and so won't be "
+                        "applied." % ", ".join(repr(app) for app in changes)
```

Comment:
Do these render in alphabetical order for consistency with `63a6a653d4282f74e35582d41ca507139c9cfbe1`?

Reply:
Good point, fixed.